src/ai/backend/common/exception.py

```diff
@@ -58,11 +58,18 @@ class InvalidImageTag(ValueError):
     ```
     """
 
+    def __init__(self, tag: str, value: str | None = None) -> None:
```

Comment:
It is hard to see what "value" means here just reading this part of the code, because "value" is a generic word and usually in exceptions like this one it indicates the value that is incorrect, i.e., "tag" here.

How about renaming it like "full_name" to reveal the intention more clearly (i.e., the contextual information where the errorneous value is taken from)?